packages/utils/telemetry-utils/src/mockLogger.ts

```diff
@@ -191,16 +199,18 @@ ${JSON.stringify(actualEvents)}`);
 		if (inlineDetailsProp && details !== undefined) {
 			assert(
 				typeof details === "string",
-				0x6c9 /* Details should a JSON stringified string if inlineDetailsProp is true */,
+				0x6_c9 /* Details should a JSON stringified string if inlineDetailsProp is true */,
```

Comment:
Add a comment that the formatting of the hex code doesn't matter here because this is an internal utility for testing? We've held on enabling the rule across the repo for that concern, right?

Reply:
I had forgotten about that. We should really just disable this rule, since it potentially causes issues. I'll get a separate PR up for that. In the meantime, I might as well just disable it locally here.

Reply:
PR: #17011